Microsoft.NET.Build.Containers/CreateNewImageToolTask.cs

```diff
@@ -176,12 +168,13 @@ protected override ProcessStartInfo GetProcessStartInfo
         string responseFileSwitch
     )
     {
+        HostObjectMagic();
```

Comment:
I would go further and inline it directly, deleting `HostObjectMagic` and making it all locals in this method.